app/react/utils/api.js

```diff
@@ -30,7 +30,11 @@ const handleErrorStatus = error => {
   } else if (error.status === 500) {
     store.dispatch(notify('An error has occurred', 'danger'));
   } else if (isNonUsualApiError(error)) {
-    store.dispatch(notify(error.json.error, 'danger'));
+    if (error.json.prettyMessage) {
+      store.dispatch(notify(error.json.prettyMessage, 'danger'));
+    } else {
+      store.dispatch(notify(error.json.error, 'danger'));
+    }
```

Comment:
perhaps a good refactor here is the much simpler:
```
store.dispatch(notify(error.json.prettyMessage || error.json.error, 'danger'));
```

Reply:
done